<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre_information</class>
 <widget class="QWidget" name="fenetre_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QLabel" name="os_name">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="os_architecture">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>521</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="processeur">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Processeur</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cpu_name">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Memoire">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MÃ©moire</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Disquedurs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Disque Durs</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Archi_proc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Architecture :</string>
   </property>
  </widget>
  <widget class="QLabel" name="proc_nbr">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>260</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Processeurs : </string>
   </property>
  </widget>
  <widget class="QLabel" name="util_proc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Utilisation : </string>
   </property>
  </widget>
  <widget class="QLabel" name="ram_util">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>En cours d'utilisation : </string>
   </property>
  </widget>
  <widget class="QLabel" name="Nombredisque">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre de disque(s) : </string>
   </property>
  </widget>
  <widget class="QLabel" name="utilisationtotale">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Utilisation totale : </string>
   </property>
  </widget>
  <widget class="QPushButton" name="actualiser">
   <property name="geometry">
    <rect>
     <x>474</x>
     <y>30</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpu_architecture">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="cpu_count">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cpu_usage">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hdd_count">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hdd_usage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>550</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ram_usage">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
